SubmitReview/ReadMe.md

```diff
@@ -0,0 +1,50 @@
+# SubmitReview
+
+Move fixed Jira tickets from one state to another using the `Request Changes` transitions.
+
+## Inputs
+
+### `github-token`
+
+Token to access the GitHub API. 
+
+### `jira-token`
+
+Token to access the Jira API.
+
+The token need the following [project permissions](https://confluence.atlassian.com/adminjiraserver/managing-project-permissions-938847145.html):
+- `Browse projects`
+- `Transition issues`
+
+## Outputs
+
+None
+
+## Prerequisites
+
+None
+
+## Example usage
+
+```yaml
+name: Submit Review
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  SubmitReview_job:
+    name: Submit Review
+    runs-on: ubuntu-latest
+    steps:
+      - id: secrets
+        uses: SonarSource/vault-action-wrapper@v3
+        with:
+          secrets: |
+            development/jira/token/{REPO_OWNER_NAME_DASH}-jira token | jira_token;
```

Comment:
I only have assumptions here, this will probably need more refinement.

Reply:
Should we use `development/team/languages` or `development/team/sonarcloud`?
I don't know if we have jira secret in the languages vault.

Reply:
Use `development/kv/data/jira`. While it doesn't exist yet, we'll ask DevInfra to create it like that, as the action is designed to be used by the entire department